src/zadaci/zadatak3/Robot.java

```diff
@@ -0,0 +1,79 @@
+package zadaci.zadatak3;
+
+import java.awt.*;
```

Comment:
haha, now this is ingenious

java.awt does indeed have Point class: https://docs.oracle.com/javase/7/docs/api/java/awt/Point.html